content/chapter-destinations/clickhouse/_index.md

```diff
@@ -157,6 +157,51 @@ You can find the available column types in the [official ClickHouse documentatio
 
 {{% include-headless "chunk/option-destination-send-timezone.md" %}}
 
+## server-side-schema()
+
+|          |                            |
+| -------- | -------------------------- |
+| Type:    | string |
+| Default: | - |
+
+By default, sending data to ClickHouse doesn't propagate the type information of the data fields. The `server-side-schema()` option provides a solution for that using the [ClickHouse format schema](https://clickhouse.com/docs/interfaces/formats#formatschema).
+
```

Comment:
[optional]

We could add that this is only needed in complex types, like `DateTime`, `LowCardinality`, etc...